core/controllers/classifier.py

```diff
@@ -123,3 +129,61 @@ def post(self):
         classifier_services.mark_training_job_complete(job_id)
 
         return self.render_json({})
+
+
+def update_failed_jobs(job_models):
+    for job_model in job_models:
+        classifier_services.mark_training_job_failed(job_model.id)
+
+
+def fetch_training_data(exp_id, state):
```

Comment:
This is not required anymore. I populate all the training data when I create the job inside the add_to_training_queue function. Its in my latest PR. So, this controller does not need to set the training data.